### ISSUE TITLE
Librarian character sprite / Skill points bug

### ISSUE THREAD

**Issue by case8180** [larsiusprime/tdrpg-bugs#1200]
Greetings,

Windows Vista 64-bit
Steam 2.0.8b - Deluxe Version (rec)

1) When I switched to standard graphics, the character sprite for the Librarian stretched off the screen. This doesn't happen for the other characters or in HD mode. If I switched to HD mode and back to standard, it didn't fix it.

![20160721072142_1](https://cloud.githubusercontent.com/assets/20578758/17022934/67d8d470-4f16-11e6-9a93-4cedfa1d2717.jpg)

2) After starting new game+ and receiving my first librarian level-up, the values for her skills no longer showed. I tried to respec her but it didn't change anything. 

![20160721073604_2](https://cloud.githubusercontent.com/assets/20578758/17023041/ee76618c-4f16-11e6-8d15-31fb8b169ae5.jpg)

**Comment by arzeth** [comment]
I have the skill points bug too.

On July 14th I reached level 18, everything was ok.
Today I continued to play the game: I completed "Zelemir's army" and leveled up (at that moment I had not completed the normal game); I wanted to upgrade Azra's skills, but all numbers in the bottom descriptions of her skills became 0. Although, when I am fighting, all numbers are correct.
Currently I am level 23, and numbers are still zero. Though, **once** numbers were ok for some reason, but then I clicked "Close" and clicked "Party" again, and numbers became 0 again.

OS: Ubuntu 16.04 x64.
Game version: Steam DQ1 2.0.8b.
Save: http://speedy.sh/dWydW/defqu-azra-lvl23-2.dfq
![Screenshot](https://i.imgur.com/It83WoO.png)

By the way, I think the "Reset pts" button should be renamed to "Reset Skills", because I was afraid to click this button because I thought it would immediately reset my HP, XP and etc.
And I'm afraid to reset skills because I don't know what's meant by "Cost": is it XP or PSI?
